src/vs/base/test/browser/formattedTextRenderer.test.ts

```diff
@@ -9,14 +9,14 @@ import { DisposableStore } from '../../common/lifecycle.js';
 import { ensureNoDisposablesAreLeakedInTestSuite } from '../common/utils.js';
 
 suite('FormattedTextRenderer', () => {
-	const store = new DisposableStore();
+	let store: DisposableStore;
 
 	setup(() => {
-		store.clear();
+		store = new DisposableStore();
 	});
 
 	teardown(() => {
-		store.clear();
+		store.dispose();
```

Comment:
FYI `ensureNoDisposablesAreLeakedInTestSuite` returns a store that handles this for you:

https://github.com/microsoft/vscode/blob/9add868cda75ab3d669f24c15d2f3b5b81a13afe/src/vs/base/parts/ipc/test/common/ipc.test.ts#L224

Reply:
I know but I try to mimic/preserve the style that's used in the test. These aren't all mine and my work has simply unveiled a bug with `ensureNoDisposablesAreLeakedInTestSuite` for which I am no paying taxes. All disposables that are created (and leaked) outside setup aren't tracked...

Reply:
no, I mean outside. Disposables that are created before the `ensure`-call aren't tracked because only that set's up the tracker.